components/Chat/ChatPage.tsx

```
@@ -316,54 +318,43 @@ export default function ClientChatPage({
-        // Submit using the hook's handleSubmit function
-        handleSubmit(e);
+    if (!hasText && !hasFiles) {
+      return;
+    }
+
+    // If this is a first message, create an optimistic chat entry in the sidebar
+    if (isFirstMessage && sidebarChatUpdater.current) {
+      // Create a better chat title based on the first message
+      let title = input.trim();
+      if (title.length > 60) {
+        title = title.substring(0, 60) + "...";
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)